modules/standard/GMP.chpl

```diff
@@ -152,7 +148,7 @@ module GMP {
   extern type __mpz_struct;
 
   /* The GMP ``mpz_t`` type */
```

Comment:
This is a documentation comment, and there are six others in the same style which just label the GMP types as their Chapel versions are defined. I'm not sure how I would expand the comment to be more descriptive, and I don't think it makes sense to remove it